src/08_comprehensions.py

```diff
@@ -27,7 +27,8 @@
 a = ["foo", "bar", "baz"]
 
 y = []
-
+for item in a:
```

Comment:
This is a for loop, which is not technically a list comprehension. Could you refactor this to be a list comprehension. If you need any help with it, reach out to me!

Reply:
Should be fixed